### DISCUSSION TITLE
exmaple-module migrations

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#587] [Q&A] [answered]
How do we generate / add new migrations to ```https://github.com/atrocore/example-module```. It seams when ```php index.php sql diff --show``` ran no migrations found even when i added ```V0Dot3Dot0.php``` class into Migrations folder. Does only the composer ```update/require``` can update/add new migrations or is there any other way to generate migrations? Appreciate your response Atrocore  team!

**Comment by rratsun** [comment, marked as answer]
``php index.php sql diff --show`` show differences between DB schema and metadata schema.

Migrations running only while system updating (composer updating).
